django_date_extensions/fields.py

```diff
@@ -17,17 +19,19 @@
 except ImportError:
     pass
 
+PREFIX_RE = re.compile('^(?i)[a-zA-Z]+[.,]?$')
```

Comment:
You are right regarding `(?i)` i will clean this up. The purpose of comma and full stop is that is a very common use case that the user use these punctuations e.g. `Approx. 2016` or 'Est. 2016' or `Approximately, 2016`. But I have other thing in mind now to further clean this up.